### ISSUE TITLE
Recently used passwords

### ISSUE THREAD

**Issue by waltertamboer** [ownpass/ownpass#9]
Browser extensions could contain a feature to show the most recently credentials. When being clicked on, the extension could open a new tab to the login page for those credentials.

**Comment by ei8fdb** [comment]
Yep, I like that feature in my current password manager. It would require the user being able to enter a URL for that website.

Ideally the password manager could grab the login page, username and password the user just used to log into the page and offer to store them in the password manager. 

Kind of like your browser will offer to do for you now, but instead of storing the user/pass in the browser, it'd do it in the manager.

Makes sense?
